### ISSUE TITLE
mutter 3.34: windows not drawn in Windows Overview or Multitasking view

### ISSUE THREAD

**Issue by worldofpeace** [elementary/gala#637]
## Describe the bug
When using Gala with Mutter 3.34, windows are visible in Multitasking View or Windows Overview.
The only visible elements for a window would be its workspace icon, and the close window button.
(I'm assuming those are independent here)

### To Reproduce
You'll need to be running gala built with mutter 3.34, which will likely imply the entire Gnome 3.34 stack. Please make sure you've also compiled wingpanel and greeter from master, which has patches for newer mutter compatibility.

1. Open a window
2. Open Windows Overview or Multitasking View
3. You can't see the windows

### Expected behavior
The windows content will be visible in both Multitasking and Overview views.

###### Multitasking View
![Screenshot from 2019-11-10 21-26-33](https://user-images.githubusercontent.com/28888242/68556896-06dfd700-0402-11ea-9347-c58aab318c82.png)

###### Show all windows/Windows Overview
![Screenshot from 2019-11-13 15-29-50](https://user-images.githubusercontent.com/28888242/68802012-1ee66f00-062b-11ea-842e-af414245e1a7.png)

### Logs
I see things like
```
.gala-wrapped[1375]: g_object_new_valist: invalid object type 'MetaShapedTexture' for value type 'ClutterActor
(gala:1622): Clutter-CRITICAL **: 20:45:35.633: _clutter_stage_queue_event: assertion 'CLUTTER_IS_STAGE (stage)' failed
(gala:1622): GLib-GObject-WARNING **: 20:45:36.446: invalid cast from 'CoglTexture2D' to '(null)'
```

in the logs. Nothing special when trying to reproduce this bug however.

## Platform Information

- OS: NixOS
- OS Version: 20.03 (development)

Gnome 3.34 stack, using wingpanel, gala, and greeter from git master.

- [x] I'm using the latest version from git that I've manually compiled
- [ ] I'm using the latest released stable version

## Additional context
I originally raised this issue in #634. 
Here's @Tireg comments:
> @worldofpeace I'm afraid I don't have enough Vala/Gobject/Clutter experience to understand what's actually responsible for drawing the objects.
The src/Widgets/WindowClone.vala seems to be the Clutter Actor responsible of the windows when you open multitasking view and windows-overview. It contains the window icon and the window close button, as well as the box responsible to catch mouse events and the window shadow.
It is contained within a WindowCloneContainer (src/Widgets/WindowCloneContainer.vala) which is responsible for placing the windows (inside a workspace) in those views.

> I don't quite get why the icon and close button are drawn as there a sibling of the WindowClone, when the actual Window is not drawn.

> It seems there is no Gala's code specific for MUTTER334 that can interfere with actual drawing operations, so I would check for Mutter changes that could lead to this issue.

> I'll probably lack some time to investigate this but I'll let you know if I find something.

This issue is not fixed in https://github.com/elementary/gala/pull/635.

I also noticed something special for Mutter 3.32, there's a screenshot of me using #566 with it and it doesn't have this issue https://github.com/elementary/gala/pull/566#issuecomment-508319393.
So it might need to be checked if this is present for 3.32, or if there's a change specific in 3.34 that we need to adapt to that caused this issue. Might look through the mutter tracker at some point.

**Comment by worldofpeace** [comment]
This could be similar to https://github.com/elementary/gala/issues/636, not sure yet.

**Comment by worldofpeace** [comment]
Ooh, i found https://gitlab.gnome.org/GNOME/mutter/issues/683 :sparkles:.

It seems to have a committed fix...

**Comment by worldofpeace** [comment]
Upon further testing, reverting that doesn't do anything. Though perhaps clutter is still the offender here.

**Comment by worldofpeace** [comment]
Fixed in  #642
